### ISSUE TITLE
Cancel, abolish, destroy the Wiki.

### ISSUE THREAD

**Issue by talaj** [mapnik/mapnik#3543]
@der-stefan [was not the first one](https://github.com/mapnik/mapnik/issues/3539#issuecomment-256782543), but I hope the last one who has got into trap of our wiki. There are more reasons why the title of this issue should become a reality:

* Large part of the content is obsolete.
* It's very often duplicate of [mapnik-reference](http://mapnik.org/mapnik-reference/).
* There is no information about existence of [mapnik-reference](http://mapnik.org/mapnik-reference/).
* Versioning of the wiki is disconnected from versioning of Mapnik.
* It's not packaged with Mapnik.
* There is also [mapnik/docs](https://github.com/mapnik/mapnik/tree/master/docs).

My plan:

1. Remove obsolete and duplicate parts.
1. Move the rest into `README.md` and [mapnik/docs](https://github.com/mapnik/mapnik/tree/master/docs). It will be tied with version of Mapnik. Anyone can edit it as simple as a wiki through the GitHub web interface.
1. Execute the wiki.

**Comment by der-stefan** [comment]
@talaj This is a very good plan. There is only one drawback: All those nice examples (a lot of them with pictures) would be lost.

**Comment by talaj** [comment]
> All those nice examples (a lot of them with pictures) would be lost.

No, those will be just moved to [mapnik/docs](https://github.com/mapnik/mapnik/tree/master/docs).

**Comment by mapsam** [comment]
@talaj happy to coordinate moving any of this to `mapnik.org/documentation` as well, as that's where we've been building node-mapnik docs for a while. It'd be nice to get everything in one central place on mapnik.org. 

Once things are moved into `/docs`, integrating into mapnik.org won't be too tough.

**Comment by springmeyer** [comment]
> - Remove obsolete and duplicate parts.
> - Move the rest into README.md and mapnik/docs. It will be tied with version of Mapnik. Anyone can edit it as simple as a wiki through the GitHub web interface.
> - Execute the wiki.

👍  good plan @talaj
